core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -149,6 +149,7 @@ public class RESTSessionCatalog extends BaseViewSessionCatalog
           .add(Endpoint.V1_RENAME_TABLE)
           .add(Endpoint.V1_REGISTER_TABLE)
           .add(Endpoint.V1_REPORT_METRICS)
+          .add(Endpoint.V1_TABLE_EXISTS)
```

Comment:
nit: I would put this entry under `Endpoint.V1_LOAD_TABLE` for grouping similar endpoints.

Reply:
+1